Xamarin.Forms.Core/FontImageSource.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms.Internals;
+using ValidateValueDelegate = Xamarin.Forms.BindableProperty.ValidateValueDelegate;
+
+namespace Xamarin.Forms
+{
+	public partial class FontImageSource : ImageSource
```

Comment:
ImageSourceConverter needs to be extended with a new format to create a FontImageSource easily from XAML. please propose something